### DISCUSSION TITLE
HVH Server

### DISCUSSION THREAD

**Discussion by DeathStrokeAMXXPRO** [GameServerManagers/LinuxGSM#3424] [Q&A] [answered]
how can i add a Command-line Parameter to disable vac security ?
CSGO SERVER

**Comment by h3o66** [comment, marked as answer]
copy the `startparameters` from the file lgsm/config-lgsm/csgoserver/_default.cfg to lgsm/config-lgsm/csgoserver/csgoserver.cfg and add the parameter to the startparameters.

The supported parameter can be found here:
https://developer.valvesoftware.com/wiki/Command_Line_Options
